### ISSUE TITLE
Passing wrong dataset when evaluating test set

### ISSUE THREAD

**Issue by ngBroken** [naderAsadi/ELM-Pytorch#1]
Hi, thanks for posting this code, looks like you are passing the wrong dataset here :

https://github.com/naderAsadi/ELM-Pytorch/blob/557a0c7866e1e1ffcb99ff018bb8d8cd008d6f74/train_elm.py#L45

You are passing the train dataset, instead of the test dataset.
